token-lending/program/src/processor.rs

```diff
@@ -1833,6 +1846,20 @@ fn get_pyth_product_quote_currency(pyth_product: &pyth::Product) -> Result<[u8;
     Err(LendingError::InvalidOracleConfig.into())
 }
 
+fn get_price(
+    switchboard_feed_info: &AccountInfo,
+    pyth_price_account_info: &AccountInfo,
+    clock: &Clock,
+) -> Result<Decimal, ProgramError> {
+    let pyth_price = get_pyth_price(pyth_price_account_info, clock).unwrap_or_default();
+    if pyth_price != Decimal::from(0u64) {
+        return Ok(pyth_price);
+    }
+    let switchboard_price = get_switchboard_price(switchboard_feed_info, clock)?;
+
+    Ok(switchboard_price)
```

Comment:
You can combine these to just `get_switchboard_price(switchboard_feed_info, clock)`
(no question mark or semicolon)